exts/betterTooltip/betterTooltip.js

```diff
@@ -100,6 +246,20 @@
                 return _measuresLabelFormatString;
             };
 
+            /**
+             * Gets or sets the format function for the label of the measures.
+             *
+             * The default format function uses the measuresLabelFormatString property for creating the label.
+             *
+             * Custom formaters receive the full tooltip model object, the measure being formated and the
```

Comment:
Custom formatters receive the full tooltip model object, the measure being formatted and the